src/relayer/Relayer.ts

```diff
@@ -916,7 +916,7 @@ export class Relayer {
         message: deposit.fromLiteChain
           ? `Deposit ${depositId} originated from over-allocated lite chain`
           : `Unable to identify a preferred repayment chain for ${originChain} deposit ${depositId}.`,
-        deposit,
+        notificationPath: "across-unprofitable-fills",
```

Comment:
It'll help if we're able to find the deposit in the block explorer, so I think we should log these fields.

```suggestion
        originChainId: deposit.originChainId,
        transactionHash: deposit.transactionHash,
        notificationPath: "across-unprofitable-fills",
```

Reply:
https://github.com/across-protocol/relayer/pull/1783/commits/7d41089b5de66f96702dbbc1de52293b74f617c5